resources/views/components/input/select/country.blade.php

```diff
@@ -1,7 +1,7 @@
 <graphql query="{ countries { two_letter_abbreviation full_name_locale } }" cache="countries">
     <div v-if="data" slot-scope="{ data }">
         <x-rapidez::input.select {{ $attributes }}>
-            <option v-for="country in Object.values(data.countries).sort((a, b) => a.full_name_locale.localeCompare(b.full_name_locale))" :value="country.two_letter_abbreviation.toUpperCase()">
+            <option v-for="country in Object.values(data.countries).filter(country => country.full_name_locale).sort((a, b) => a.full_name_locale.localeCompare(b.full_name_locale))" :value="country.two_letter_abbreviation.toUpperCase()">
```

Comment:
We're using `toSorted()` at the region select; `Object.values().sort()` here, cleaner to do the same everywhere?

Reply:
Should be `toSorted()` everywhere, `sort()` changes the array in place and can cause weird reactivity issues. Not here, of course, because it's behind an `Object.values()`, but it's good to keep it consistent this way.

Reply:
But I found multiple polyfills for `toSorted` in projects, sure this is the smartest option to use? We don't have that in the core?

Reply:
We could add a polyfill. However, note that if we're migrating to Tailwind v4, any browsers that don't support `toSorted` also don't support Tailwind v4 styling.